### DISCUSSION TITLE
How can I resolve a "spawnSync cdk ENOENT" error when using integ-runner?

### DISCUSSION THREAD

**Discussion by RiccoA** [aws/aws-cdk#25274] [Q&A]
I'm trying to use the integ-runner library but I haven't been able to do a successful snapshot or test with any cdk project. I suspect it's something in my environment.

I am running the tests in this project: [https://github.com/aws-samples/cdk-integ-tests-sample](https://github.com/aws-samples/cdk-integ-tests-sample).

And this is the error I get:
`Error during integration test: Error: spawnSync cdk ENOENT`

My cdk synth and cdk deploy both are working correctly and I have docker installed, and I'm using windows 10.

**Comment by maxscn** [comment]
Also running into this, using Windows 11. It does succeed to run the test using Github Actions and on ubuntu-latest, so likely due to my machine. Have both esbuild and docker installed. 

Did you manage to solve it @RiccoA?
